cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -109,6 +111,16 @@ the cloud specific control loops shipped with Kubernetes.`,
 
 // Run runs the ExternalCMServer.  This should never exit.
 func Run(c *cloudcontrollerconfig.CompletedConfig, stopCh <-chan struct{}) error {
+	ctx, cancel := context.WithCancel(context.TODO())
+	defer cancel()
```

Comment:
cancel is called too many times, pretty sure duplicate with line 166

Reply:
fixed